### ISSUE TITLE
Disable "default editor"

### ISSUE THREAD

**Issue by sarmamen** [fman-community/fman#783]
I am using windows. When creating a new file pops the following diag:
![image](https://user-images.githubusercontent.com/18058256/98661388-ad78f380-2346-11eb-8248-f37a69349db2.png)

After pressing OK above, appears the following diag:
![image](https://user-images.githubusercontent.com/18058256/98661506-d00b0c80-2346-11eb-8c31-c15b8180c54b.png)

My question is: "is there any way to disable the second diag? I dont want to select a default editor and it is annoying to press Cancel each time".

The reason for that is that i wrote myself a small "open with" plugin and by pressing CTRL+R i will get a list of editors i can chose from:
![image](https://user-images.githubusercontent.com/18058256/98662096-938be080-2347-11eb-937d-50c3b66607b5.png)

When everything is running ok i will make the plugin public :)

**Comment by mherrmann** [comment]
As the editor in the second dialog, you could select an .exe that simply does nothing. Would that work?

**Comment by sarmamen** [comment]
I tried it out, and it is working. I tried different FileTypes (.txt, .json, .py, ...)

I also tried an empty file without extension and at first got this error. 
I tried the same thing 5 times after and the error **didn't** appear. Strage.
But the most important thing is that i got rid of the PopUp.
Thank you.
![image](https://user-images.githubusercontent.com/18058256/98919470-22773500-24cf-11eb-9554-3aa91a4b537e.png)
